package.yaml

```diff
@@ -32,6 +32,8 @@ tests:
   typed-process-test:
     main: Spec.hs
     source-dirs: test
+    build-tools:
+    - hspec-discover:hspec-discover >=2 && <3
```

Comment:
I don't understand why this is needed.

Reply:
I added in the case one wants to run the tests with cabal:
```shell
hpack
cabal test all
```
Without the generated `build-tools-depends` entry in the Cabal file you get a `could not execute: hspec-discover`. For stack it is indeed not necessary.